backend/spec/models/question_spec.rb

```diff
@@ -76,4 +76,20 @@
       expect(question.errors.full_messages).to match(['Ready to be answered Has to be "ready to be answered" when survey is ready.'])
     end
   end
+
+  describe 'validate survey same user' do
+    it 'is valid when question and survey have the same user' do
+      survey_teacher = create(:survey, user: user_teacher)
+      question = create(:question, user: user_teacher, survey: survey_teacher)
+
+      expect(question).to be_valid
+    end
+
+    it 'is not valid when question and survey have different user' do
```

Comment:
add spec to check the error message in the model

Reply:
Ok